packaging/linux/common/khiops_env.in

```diff
@@ -79,16 +82,29 @@ if [ -f "/usr/share/khiops/khiops.jar" ]; then
     fi
 fi
```

Comment:
This `if` here inadvertently sets `KHIOPS_CLASSPATH` and `KHIOPS_JAVA_PATH` in Conda environments if:
- Java is natively installed, *and*
- previous Khiops installation is present.

However, OpenJDK can also be installed in Conda environments (from `conda-forge`). Thus, settings these variables in `khiops_env` (which, unlike the `khiopsè and `khiops_coclustering` scripts, is used by "anybody" - including the Python library) to inadvertent values could lead to confusion at the least IMHO.